torchgen/api/python.py

```diff
@@ -1109,6 +1109,7 @@ def dispatch_lambda_arg(cpp_arg: Binding) -> DispatchLambdaArgument:
     "::std::tuple<at::Tensor,at::Tensor,at::Tensor>",
     "::std::tuple<at::Tensor,at::Tensor,at::Tensor,at::Tensor>",
     "::std::tuple<at::Tensor,at::Tensor,at::Tensor,at::Tensor,at::Tensor>",
+    "::std::tuple<at::Tensor,at::Tensor,at::Tensor,at::Tensor,at::Tensor,at::Tensor>",
```

Comment:
The comment above says we need to make modifications to `wrap_outputs.h`, should we be doing that? 

Adding a new return type is fine in itself. Saving another tensor for backward could potentially be more problematic, so we should only be doing so if it is necessary (maybe it is, I haven't read `RnnOps.mm`, so I can't comment on that).

Reply:
I checked `wrap_outputs.h`, it seems to support any number of `Tensor` already through variardic templates

Reply:
Also, worth noting that all data from the added tensor is used in backdrop and needed for correctness. So, yes, it can be considered necessary